### PR TITLE
Updated weapon roll window and bugg fix

### PR DESCRIPTION
**Pull request by Batziy**
image

### CODE REVIEW

**Comment by AnthonyMonette** [comment]
It seems that there is some conflicts, with latest version

File: module/sheets/WitcherActorSheet.js [hunk 1]
```diff
@@ -2185,52 +2126,30 @@ export default class WitcherActorSheet extends ActorSheet {
...
-      if (this.actor.type =="character" && isMeleeAttack){ 
-        content += `<label>${game.i18n.localize("WITCHER.Dialog.attackMeleeBonus")}: ${this.actor.system.attackStats.meleeBonus} </label><br />`
-      }
-
+      let ammunitions = ``
+      let noAmmo = 0
+      let ammunitionOption = ``
       if (item.system.usingAmmo){
-        let ammunitions = this.actor.items.filter(function(item) {return item.type=="weapon" &&  item.system.isAmmo});
+        ammunitions= this.actor.items.filter(function(item) {return item.type=="weapon" &&  item.system.isAmmo});
```
Comment (AnthonyMonette) [inline module/sheets/WitcherActorSheet.js:2133]: add space before = and it seems that after  && it has an unessessary space

File: module/sheets/WitcherActorSheet.js [hunk 2]
```diff
@@ -2185,52 +2126,30 @@ export default class WitcherActorSheet extends ActorSheet {
...
+              ammunitionOption += `<option value="${element._id}"> ${element.name}(${element.system.quantity}) </option>`;
           });
-          content += ` <label>${game.i18n.localize("WITCHER.Dialog.Ammunation")}: <select name="ammunation">${ammunationOption}</select></label> <br /><br />`
         }
       }
-              
+      
+      let Mymelebonus=this.actor.system.attackStats.meleeBonus
+      let data = {item, attackSkill, displayDmgFormula, isMeleeAttack, noAmmo, ammunitionOption, ammunitions, Mymelebonus}
+      const myDialogOptions ={width: 500}
```
Comment (AnthonyMonette) [inline module/sheets/WitcherActorSheet.js:2147]: space after  =

File: module/sheets/WitcherActorSheet.js [hunk 3]
```diff
@@ -2300,7 +2218,7 @@ export default class WitcherActorSheet extends ActorSheet {
                 if (isRicochet) { attFormula += "-5"; }
                 if (isBlinded) { attFormula += "-3"; }
                 if (isSilhouetted) { attFormula += "+2"; }
-                if (isAiming) { attFormula += `+${customAim}`}
+                if (customAim>0) { attFormula += `+${customAim}`}
```
Comment (AnthonyMonette) [inline module/sheets/WitcherActorSheet.js:2221]: surround > with spaces

File: module/sheets/WitcherActorSheet.js [hunk 4]
```diff
@@ -2517,7 +2435,7 @@ export default class WitcherActorSheet extends ActorSheet {
             }
           }
         }
-      }).render(true)  
+      },myDialogOptions).render(true)
```
Comment (AnthonyMonette) [inline module/sheets/WitcherActorSheet.js:2438]: space after the ","

**Comment by AnthonyMonette** [comment]
I love the new visuals great work 👍

File: module/sheets/WitcherActorSheet.js (hunk 1 above)
Comment (Batziy) [reply module/sheets/WitcherActorSheet.js]: Fixed

File: module/sheets/WitcherActorSheet.js (hunk 2 above)
Comment (Batziy) [reply module/sheets/WitcherActorSheet.js:2147]: Fixed

File: module/sheets/WitcherActorSheet.js (hunk 3 above)
Comment (Batziy) [reply module/sheets/WitcherActorSheet.js]: Fixed

File: module/sheets/WitcherActorSheet.js (hunk 4 above)
Comment (Batziy) [reply module/sheets/WitcherActorSheet.js]: Fixed

**Comment by Batziy** [comment]
> I love the new visuals great work 👍

Thx :) there will be more ^^

**Pull request merged**